ADOPTERS.md

```diff
@@ -0,0 +1,11 @@
+This is an alphabetical list of known adopters of Vitess. Some have already gone into production, and others are at various stages of testing.
+
+* [BetterCloud](http://bettercloud.com)
+* [FlipKart](http://flipkart.com)
+* [HubSpot](http://hubspot.com)
```

Comment:
Can you link to http://product.hubspot.com/ instead? That's the landing page for our dev team blog and such.